trino/etc/catalog/pravega.properties

```diff
@@ -0,0 +1,3 @@
+connector.name=pravega
+pravega.controller=tcp://10.247.179.147:9000
```

Comment:
we should use localhost here instead of "real" ip

Reply:
Fixed.